client/src/data/QuizMockData.json

```diff
@@ -0,0 +1,70 @@
+{
+  "_id": "quiz456",
+  "_slug": "sdg-14-quiz",
```

Comment:
Is er een reden dat we het _slug noemen ipv slug? 
_id is omdat mongodb dat zelf zo genereert

Reply:
Nee niet echt, mark had het zo erin gelaten. En ik wilde niet zomaar de properties veranderen als andere developers nog uitgaan van deze json. Ik kan het wel weer veranderen naar slug ipv _slug